arch-makepkg.md

```diff
@@ -0,0 +1,50 @@
+# Installing SystemVerilog on Arch-based systems using Pacman and makepkg
+
+In this guide we'll show you how to install all necessary tools for synthesizing or simulating your SystemVerilog projects.
+
+> "[makepkg](https://git.archlinux.org/pacman.git/tree/scripts/makepkg.sh.in) is a script to automate the
+> building of packages. The requirements for using the script are
+> a build-capable Unix platform and a [PKGBUILD](https://wiki.archlinux.org/index.php/PKGBUILD). 
+> ([Arch Wiki](https://wiki.archlinux.org/index.php/Makepkg))
+
+## Installing SystemVerilog toolchain
+
+This guide has been tested on *5.10.6-arch1-1 x86_64 GNU/Linux*, but should work for all Arch-based systems.
+
+### Getting the PKGBUILD and the SystemVerilog zip archive.
+Either clone or download and unzip this repository.
+
+You should have received a SystemVerilog zip archive:
+Place the archive in the repository's directory.
+**Don't** unzip it.
+Rather make the name all lowercase,
+remove all spaces and the part in parenthesis.
+E.g. `SystemVerilog (Synthese und Simulation)-20210116.zip` becomes `systemverilog-20210116.zip`.
```

Comment:
This timestmp `20210116` does not represent the version of the SystemVerilog tools instead it's just a timestamp when you've downloaded the archive. That's why I recommend we just use a fixed version like `1`.
In this case  we can also just change the name of the archive to `systemverilog.zip` or?